### ISSUE TITLE
Make exe file location configurable

### ISSUE THREAD

**Issue by mmarseu** [sakai135/wsl-vpnkit#104]
At my company, we're using AppLocker to lock down the executables people can run on their Windows machines. There are just some limited paths where I'm allowed to run things. The user profile folder isn't one of those locations unfortunately.

Would it be possible to make the location for executables on the windows host system configurable?

**Comment by mmarseu** [comment]
Another thing: If that could be added, could it also be backported to 2.x? I can't use 3.x because McAfee considers `wsl-gvproxy.exe` to be a trojan.

**Comment by sakai135** [comment]
In v0.3.x, you can copy `wsl-gvproxy.exe` to another location in the Windows side, then set `GVPROXY_PATH` to that path in the config file at `youruserprofiledir/wsl-vpnkit/wsl-vpnkit.conf`.

In v0.2.x, you can set `VPNKIT_PATH` in the config file.

**Comment by mmarseu** [comment]
Thank you!

I'm going to expand a little on your comment, so others like me - let's say less-than-enlightened people ;) - might not fall into the same traps I did along the way.
I had to set `VPNKIT_PATH` and `NPIPERELAY_PATH` for version 0.2.5. The paths are evaluated inside WSL, so they must be WSL paths, not Windows paths.

Example `$USERPROFILE\wsl-vpnkit\wsl-vpnkit.conf`:
```
VPNKIT_PATH=/mnt/C/wsl-vpnkit/vpnkit.exe
NIPERELAY_PATH=/mnt/C/wsl-vpnkit/npiperelay.exe
```
